### ISSUE TITLE
Duplicate auto import suggestions for traits

### ISSUE THREAD

**Issue by lancelote** [intellij-rust/intellij-rust#4501]
## Environment

* **IntelliJ Rust plugin version:** 0.2.107.SNAPSHOT-192-dev
* **Rust toolchain version:** 1.40.0-nightly (787005079 2019-10-04) x86_64-apple-darwin
* **IDE name and version:** CLion 2019.2.2 (CL-192.6603.37)
* **Operating system:** macOS 10.14.6

## Problem description

Plugin suggests duplicate items in the completion for a trait:

<img width="391" alt="Screen Shot 2019-10-06 at 17 43 08" src="https://user-images.githubusercontent.com/2500320/66288642-a7624900-e8e2-11e9-9902-78adfcf0c25e.png">

## Steps to reproduce

```rust
#[derive(Debug)]
struct Foo;

fn main() {
    println!("{:?}", 1);

    assert_eq!(1, 2);

    1.fmt();
}
```

Call "Import" intention action on "fmt".

**Comment by Undin** [comment]
Looks like experimental macro expansion engine is a necessary part to reproduce it

**Comment by dima74** [comment]
@lancelote could you please check, looks like it is fixed in latest nightly 0.4.161.4247-212-nightly

**Comment by lancelote** [comment]
Works fine for me indeed, thank you 👍
